docs/admin/cluster-management.md

```diff
@@ -22,41 +22,33 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Management
 
```

Comment:
Can you add an introductory sentence for the doc right after the "Cluster Manaement" heading. Maybe "This document describes several topics related to cluster management: upgrading your cluster's master and nodes, performing node maintenance, and upgrading the API version of your cluster."